README.md

```diff
@@ -2133,6 +2150,21 @@ docker run -d \
   scality/s3utils service-level-sidecar/index.js
 ```
 
+#### Scuba
+
+The scuba backend can be enabled by setting `SIDECAR_ENABLE_SCUBA`.
```

Comment:
what are sample values for these?
whats 't' in `SIDECAR_ENABLE_SCUBA=t` its not that obvious, so may be examples would help

Reply:
If it's a boolean I'd go for `1` instead of `t`

Reply:
It can be any one of these (after `toLowerCase()`)
```js
    truthy: new Set([
        'true',
        'on',
        'yes',
        'y',
        't',
        'enabled',
        'enable',
        '1',
    ]),
```
I'll change the example to `true` to be more clear